packages/_private/vanilla-demo/vite.config.ts

```diff
@@ -70,6 +70,10 @@ export const withSpriteSheetGenerator = {
         'home',
         'contact_mail',
         'contact_emergency',
+        'grid_view',
+        'wallpaper',
+        'house',
+        'chat',
```

Comment:
The icons of the alerts are missing and are therefore not visible:
![image](https://github.com/user-attachments/assets/67c3e5a4-c908-49b5-9852-2281e8ba8ab6)

```suggestion
        'grid_view',
        'wallpaper',
        'house',
        'chat',
        'info',
        'check_circle',
        'settings',
        'warning',
        'error',
```

Reply:
I see, will adjust this

Reply:
Fixed